yarn-project/end-to-end/src/e2e_non_contract_account.test.ts

```diff
@@ -81,6 +81,8 @@ describe('e2e_non_contract_account', () => {
     // Add the note
     const note = new Note([new Fr(value)]);
 
+    // We have to manually add the note because the note ciphertext is encrypted with zero keys since there
```

Comment:
See comment earlier related to the test_contract. It is not actually encrypted, it is not broadcast.

Reply:
Good point. Clarified in https://github.com/AztecProtocol/aztec-packages/pull/6741/commits/c998e604096d57f45861528c9bc3f8f4633715ee